openfisca_us/variables/gov/states/mo/tax/income/deductions/mo_federal_income_tax_deduction.py

```diff
@@ -0,0 +1,38 @@
+from openfisca_us.model_api import *
+
+
+class mo_federal_income_tax_deduction(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "MO Federal income tax deduction"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "https://dor.mo.gov/forms/MO-1040%20Instructions_2021.pdf#page=8",
+        "https://revisor.mo.gov/main/OneSection.aspx?section=143.171&bid=49937&hl=federal+income+tax+deduction%u2044",
+    )
+    defined_for = StateCode.MO
+
+    def formula(tax_unit, period, parameters):
+        mo_adjusted_gross_income = tax_unit("mo_adjusted_gross_income", period)
+        federal_tax = tax_unit("income_tax", period)
```

Comment:
I think this needs the income tax variable that assumes no state income tax to avoid the circular reference. @nikhilwoodruff what's that variable called?